PaddleCV/object_detection/ppdet/utils/create_list.py

```diff
@@ -0,0 +1,73 @@
+import os
```

Comment:
1. Put this to  `object_detection/dataset/voc/ ` ?  also add the data download script ?

2. Add copyright